lib/Optimizer/Transforms/GenKernelExecution.cpp

```diff
@@ -29,8 +29,8 @@ using namespace mlir;
 
 namespace {
 // Define some constant function name strings.
-static constexpr const char cudaqRegisterLambdaName[] =
-    "cudaqRegisterLambdaName";
+static constexpr const char cudaqRegisterCallableName[] =
+    "cudaqRegisterCallableName";
```

Comment:
I think you can skip this one. This is actually for giving an anonymous C++ lambda expression a unique name so we can find it at JIT time.